unittests/InstructionCountCI/AVX128/VEX_map1.json

```diff
@@ -0,0 +1,6318 @@
+{
+  "Features": {
+    "Bitness": 64,
+    "EnabledHostFeatures": [],
+    "DisabledHostFeatures": [
+      "FCMA",
+      "RPRES",
+      "AFP",
+      "FLAGM",
+      "FLAGM2",
```

Comment:
Could we enable flagm for the avx128 instcountci? It should save a few instructions on vptest.

(I've considered removing non-flagm instcountci. Even if we're not going to raise our minspec for FEX doesn't mean we need to have instcountci for it... But that's a tangential discussion to be had)

Reply:
Added the same flagm map2 file that we had for SVE256, but this time for asimd

Reply:
v(u)comiss notably missing

Reply:
You're right, that was in the other map file. Added that one as well.

Reply:
Looks good, thx